force-app/main/default/classes/CountTasksTest.cls

```diff
@@ -0,0 +1,107 @@
+@isTest
+public class CountTasksTest {
+    /*Apex Test Class for all scenarios(insert, update, delete, close task, re open task)*/
```

Comment:
Class Headers should go at the top. There are great extensions around for [Apex Docs](https://marketplace.visualstudio.com/items?itemName=PeterWeinberg.apexdox-vs-code)